charts/areaCharts/StackedArea.tsx

```diff
@@ -351,6 +344,7 @@ export class StackedArea extends React.Component<{
 
         return (
             <Tooltip
+                tooltipOwner={this.props.chart}
```

Comment:
`tooltipOwner` is just a temporary step to something cleaner. Passing this param is a slight improvement over the current global (and allows the story to work) but we can clean that up later.